### PR TITLE
[LLM Doc] Add optimize_model doc in transformers api

### PR DESCRIPTION
**Pull request by sgwhat**
## Description
Added missing API documentation for the transformers-format API in this PR.

### 1. Why the change?
`from_pretrained` only has args and kwargs in the implementation, need to add it manually in the doc.

### 2. User API changes
N/A

### 3. Summary of the change 

Document missing args in `from_pretrained` and `load_low_bit`.

### 4. How to test?
- [x] Document test

https://optimize-model-doc.readthedocs.io/en/latest/doc/PythonAPI/LLM/transformers.html.

### CODE REVIEW

File: python/llm/src/bigdl/llm/transformers/model.py [hunk 1]
```diff
@@ -54,13 +54,14 @@ def from_pretrained(cls,
...
 
-        Two new arguments are added to extend Hugging Face's from_pretrained method as follows:
+        Three new arguments are added to extend Hugging Face's from_pretrained method as follows:
 
         :param load_in_4bit: boolean value, True means load linear's weight to symmetric int 4.
         :param load_in_low_bit: str value, options are sym_int4, asym_int4, sym_int5, asym_int5
                                 or sym_int8. sym_int4 means symmetric int 4, asym_int4 means
                                 asymmetric int 4, etc. Relevant low bit optimizations will
                                 be applied to the model.
+        :param optimize_model: boolean value, Whether to further optimize llm model.
```
Comment (jason-dai) [inline python/llm/src/bigdl/llm/transformers/model.py:64]: default to True

File: python/llm/src/bigdl/llm/transformers/model.py [hunk 2]
```diff
@@ -54,13 +54,18 @@ def from_pretrained(cls,
         Load a model from a directory or the HF Hub. Use load_in_4bit or load_in_low_bit parameter
         the weight of model's linears can be loaded to low-bit format, like int4, int5 and int8.
 
-        Two new arguments are added to extend Hugging Face's from_pretrained method as follows:
+        Three new arguments are added to extend Hugging Face's from_pretrained method as follows:
 
         :param load_in_4bit: boolean value, True means load linear's weight to symmetric int 4.
+                             Default to False.
```
Comment (hkvision) [inline python/llm/src/bigdl/llm/transformers/model.py:60]: to be?

File: python/llm/src/bigdl/llm/transformers/model.py [hunk 3]
```diff
@@ -54,13 +54,18 @@ def from_pretrained(cls,
...
         :param load_in_low_bit: str value, options are sym_int4, asym_int4, sym_int5, asym_int5
                                 or sym_int8. sym_int4 means symmetric int 4, asym_int4 means
                                 asymmetric int 4, etc. Relevant low bit optimizations will
                                 be applied to the model.
+        :param optimize_model: boolean value, Whether to further optimize llm model.
```
Comment (hkvision) [inline python/llm/src/bigdl/llm/transformers/model.py:65]: to further optimized the *low_bit* llm model?

File: python/llm/src/bigdl/llm/transformers/model.py [hunk 4]
```diff
@@ -54,13 +54,18 @@ def from_pretrained(cls,
...
+                               Default to True to optimize the model.
```
Comment (hkvision) [inline python/llm/src/bigdl/llm/transformers/model.py:66]: Just `Default to be True`?

File: python/llm/src/bigdl/llm/transformers/model.py [hunk 5]
```diff
@@ -128,6 +133,15 @@ def load_low_bit(cls,
                      pretrained_model_name_or_path,
                      *model_args,
                      **kwargs):
+        """
+        Load a low bit optimized model (including INT8, INT5 and INT4) from a saved ckpt.
```
Comment (hkvision) [inline python/llm/src/bigdl/llm/transformers/model.py:137]: reorder to INT4, INT5 and INT8?

**Review by hkvision — APPROVED** [review]

**Pull request merged**